Umbraco-Cloud/Set-Up/Users-On-Cloud/index.md

```diff
@@ -10,6 +10,22 @@ On Umbraco Cloud, users work in almost the same way as on a normal installation
 
 In this article, we will show how users work, as well as explain the different settings for users on Umbraco Cloud.
 
+## Umbraco ID
+
+On Umbraco CLoud project we use Umbraco ID for new projects and projects on 8.9.1 and above.
+
+Umbraco ID is a centralised login for all users on Umbraco Cloud, both team members and Umbraco Backoffice users.
+It is used when you log in to the Umbraco Cloud Portal, projects,
+as well as when you clone down a project to your local machine.
+
+Because of this there are no longer any user files added in the data folder when a user is invited to a specific project.
```

Comment:
I don't think this is necessary to mention. This is not really something that should be a concern for our users/developers,